### ISSUE TITLE
icons issue

### ISSUE THREAD

**Issue by 1arsi1** [kristofhracza/tim_apple#7]
you've got a deagle repeating itself (https://github.com/kristofhracza/tim_apple/blob/master/util/weaponInfo.hpp).
![image](https://github.com/kristofhracza/tim_apple/assets/61199844/b26ef52a-ed9b-4293-a1a7-42873b693a10)

**Comment by 1arsi1** [comment]
![image](https://github.com/kristofhracza/tim_apple/assets/61199844/58bafbbb-0964-48f1-ae72-e76dd54c514b)
repeats

**Comment by 1arsi1** [comment]
![image](https://github.com/kristofhracza/tim_apple/assets/61199844/b78fdb9e-7249-4c4d-abeb-2b4052adfccb)
and SSG-08 is still not showing up, is it working for you?

**Comment by 1arsi1** [comment]
![image](https://github.com/kristofhracza/tim_apple/assets/61199844/165b325d-c280-4804-af0d-8993aa06d109)
on bot work..

**Comment by kristofhracza** [comment]
I fixed the `deagle` issue [here](https://github.com/kristofhracza/tim_apple/commit/2181c19cadecaf987c6f856cc6b0bca39f90dd4d)

Though it wasn't causing issues during gameplay as the initial values was later overwritten.

As mentioned [here](https://github.com/kristofhracza/tim_apple/issues/5) I don't know what's causing the `ssg` not to appear.
As of right now I will close this issue and will not respond to any other regarding this problem, as I've already answered it.
